bin/singularity.in

```diff
@@ -22,6 +22,8 @@
 # 
 # 
 
+# Relax restrictive umasks that can cause several problems
+umask 0002
```

Comment:
Change to `umask 0022` and we'll call it good yeah? :grinning: 

(Don't forget the changelog entry)

Reply:
Concur with @jmstover here - since 0022 is a default in EL systems (and still works) we should prefer that over the more lenient Ubuntu default 0002.